app/controllers/press-releases/press-release.js

```diff
@@ -57,11 +57,11 @@ export default class PressReleasesPressReleaseController extends Controller {
 
   @task
   *publish(publicationDate) {
-    if (!publicationDate) {
+    if (!(publicationDate instanceof Date)) {
       publicationDate = new Date();
     }
 
-    let publicationEvent = yield this.model.publicationEvent;
+    let publicationEvent = yield this.model.pressReleasePublicationEvent;
```

Comment:
Zie ook comments bij snapshot class. Het publication event wordt niet gesnapshot, dus hier moet je de relatie van het persbericht ophalen via `this.snapshot.pressRelease.publicationEvent`.